test-suite/src/ast_builder/function/text/position_and_indexing.rs

```diff
@@ -18,7 +18,7 @@ test_case!(position_and_indexing, async move {
     let actual = table("Item")
         .create_table()
         .add_column("id INTEGER PRIMARY KEY")
-        .add_column("index TEXT")
+        .add_column("index INTEGER")
```

Comment:
Since this is a test module for text-related functions, it would make sense to either make it text, or change the module location.